cinema/tests/test_movie_api.py

```diff
@@ -93,9 +91,7 @@ def test_filter_movies_by_genres(self):
 
         movie3 = sample_movie(title="Movie without genres")
 
-        res = self.client.get(
-            MOVIE_URL, {"genres": f"{genre1.id},{genre2.id}"}
-        )
+        res = self.client.get(MOVIE_URL, {"genres": f"{genre1.id},{genre2.id}"})
```

Comment:
Ensure that the filtering logic for genres in the API is correctly implemented to handle multiple genre IDs. The test seems to assume that the API can filter movies by multiple genres, so verify that this functionality is supported.